pom.xml

```diff
@@ -25,6 +26,8 @@
         <c3p0Version> 0.9.5.2 </c3p0Version>
         <springVersion> 4.3.1.RELEASE </springVersion>
         <javaVersion> 1.8 </javaVersion>
+        <hiberVersion> 5.2.1.Final </hiberVersion>
```

Comment:
Fixed. c3p0 is an artifactId of a JDBC Connection pooling / Statement caching library com.mchange/c3p0. Chaned to comMchangeC3p0Version.